assets/css/__style-guide.css

```diff
@@ -1,20 +1,20 @@
 :root {
-  --primary: #13a3fd;
-  --primary-light: #ffffff;
-  --primary-dark: #141414;
-  --text-white: #ffffff;
-  --text-grey: #8d8d8d;
+  --colors-primary: #13a3fd;
+  --theme-light: #ffffff;
+  --theme-dark: #181818;
+  --colors-white: #ffffff;
+  --colors-grey: #8d8d8d;
   --colors-green: #8fff00;
   --colors-yellow: #ffc226;
   --colors-orange: #ff6826;
-  --font-title: "Montserrat", sans-serif;
-  --font-body: "Sora", sans-serif;
-  --dark-theme-bg: #181818;
+  --font-montserrat: 'Montserrat', sans-serif;
+  --font-sora: 'Sora', sans-serif;
   --breakpoint-mobile: 767px;
   --breakpoint-tablet: 1023px;
   --breakpoint-laptop-and-above: 1024px;
   --max-container-width: 108em;
-  --container-width: 90%;
+  --container-width: 80%;
+  --sm-container-width: 90%;
```

Comment:
Since these `container` vars will not be used anywhere else but in the `VContainer` component, should we remove them from here?